docs/release-notes/release-notes-0.14.2.md

```diff
@@ -18,14 +18,14 @@
 
 ### Functional Changes/Additions
 
+* [`litcli`: global flags to allow overrides via environment variables](https://github.com/lightninglabs/lightning-terminal/pull/1007) `lncli` --help for more details.
+
```

Comment:
Could you please break lines to a 80-char wrapping? This also mentions `lncli`, which should be `litcli`. Please also squash both commits (to follow our contribution guidelines https://github.com/lightningnetwork/lnd/blob/master/docs/code_contribution_guidelines.md), thank you!

Reply:
fixed. Sorry about that.

Reply:
no worries :pray:, this still mentions `lncli --help` instead of `litcli`